gofmt.go

```diff
@@ -42,6 +42,10 @@ var (
 	langVersion = flag.String("lang", "", "target Go version in the form 1.X (default from go.mod)")
 	extraRules  = flag.Bool("extra", false, "enable extra rules which should be vetted by a human")
 	showVersion = flag.Bool("version", false, "show version and exit")
+
+	// DEPRECATED
+	rewriteRule = flag.String("r", "", "DEPRECATED: you can use gofmt for it instead")
+	simplifyAST = flag.Bool("s", false, "DEPRECATED: the simplify flag is enabled by default")
```

Comment:
Showing "DEPRECATED" lines in `gofumpt -h` doesn't feel great. We should just not show these flags to new users at all.

I think the solution is to roll our own printing of declared flags, like in https://github.com/mvdan/sh/blob/f965c621ea14199cad039642cec49b2e0626aa70/cmd/shfmt/main.go#L76. Note that, since the usage text is written by hand, the usage line for each declared flag can then be an empty string to avoid duplication.

You can just copy-paste the current help text generated by the flag package as a start, minus the two deprecated flags; we can refine it later on.

Reply:
made the changes, PTAL